vmm/src/api/http.rs

```diff
@@ -161,6 +161,8 @@ lazy_static! {
         r.routes.insert(endpoint!("/vm.send-migration"), Box::new(VmActionHandler::new(VmAction::SendMigration(Arc::default()))));
         r.routes.insert(endpoint!("/vm.shutdown"), Box::new(VmActionHandler::new(VmAction::Shutdown)));
         r.routes.insert(endpoint!("/vm.snapshot"), Box::new(VmActionHandler::new(VmAction::Snapshot(Arc::default()))));
+        #[cfg(feature = "gdb")]
```

Comment:
Probably a good idea to rename this as `guest_debug`. @retrage Do you see any problem with that?

Reply:
It's reasonable to rename the feature flag as the coredump feature is not a part of GDB support, but it shares debugger-independent code.

Reply:
Do you mean move coredump_xxx from Debuggable trait into a new trait which may be GuestDebuggable? If so, I will move all the new code in gdb.rs into a new separate module coredump.

Reply:
@up2wing There are two things:

1. Using a different name for the feature flag e.g. `#[cfg(feature = "gdb")]`-> `#[cfg(feature = "guest_debug")]` and 
2. Putting your new code in a new module rather than putting it in the gdb module.

Reply:
Or use a dedicated `coredump` feature to gate so that users can decide to include one or both.